src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingGuards.java

```diff
@@ -427,8 +427,12 @@ public void removeCitizen(final CitizenData citizen)
             if (optCitizen.isPresent())
             {
                 optCitizen.get().removeAllHealthModifiers();
-
-                optCitizen.get().getEntityAttribute(SharedMonsterAttributes.ARMOR).setBaseValue(SharedMonsterAttributes.ARMOR.getDefaultValue());
+                optCitizen.get().setItemStackToSlot(EntityEquipmentSlot.CHEST, ItemStackUtils.EMPTY);
```

Comment:
Does this move the item to his normal inventory?
Or does this delete it?

Reply:
i can answer this. the armour is equipped while keeping the item in his inventory, same with tools and etc. as such there's technically two copies of the same items (one in armor slot and one in inventory) . but yes i had him test this and it keeps the items in his inventory, while removing them from his "armor slots"

Reply:
The items never leave his inventory. if you remove the armor from his inventory he will remove it, and all this does is force him to remove the armor. I tested it on my end and it works like expected.